containers/sprm-to-json/context/main.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+from glob import glob
+from pathlib import Path
+from os import makedirs
+import json
+
+from shapely.geometry import Polygon
+import pandas as pd
+import numpy as np
+
+# Number of vertices for the polygon boundary data.
+NUM_VERTICES = 20
+GENES_FILE_SUFFIX = ".ome.tiff-cell_channel_mean.csv"
+CLUSTER_FILE_SUFFIX = ".ome.tiff-cell_cluster.csv"
+POLYGON_FILE_SUFFUX = ".ome.tiff-cell_polygons_spatial.csv"
+
+
+def sprm_to_items(input_file):
+    df = pd.read_csv(input_file)
+    df = df.set_index("ID")
+    df_items = df.T.to_dict().items()
+    return df_items
+
+
+def write_genes_or_factors_to_cells(input_file, cells={}, genes=True):
+    vitessce_key = "genes" if genes else "factors"
+    df_items = sprm_to_items(input_file)
+    for (cell_key, value) in df_items:
+        cells[cell_key][vitessce_key] = {}
+        for value_key in value:
+            cells[cell_key][vitessce_key][value_key] = (
+                value[value_key] if genes else str(value[value_key])
+            )
+    return cells
+
+
+def write_polyon_bounds(input_file, cells):
+    df_items = sprm_to_items(input_file)
+    for (k, v) in df_items:
+        shape = eval(v["Shape"])
+        poly = Polygon(shape)
+        idx = np.round(np.linspace(0, len(shape) - 1, NUM_VERTICES + 1)).astype(int)
+        shape_downsample = np.asarray(shape)[idx]
+        poly_downsample = Polygon(shape_downsample)
+        cells[k] = {
+            "xy": list(poly.centroid.coords)[0],
+            "poly": list(poly_downsample.exterior.coords),
+        }
+
+    return cells
+
+
+def create_cells(tile_str, input_dir, output_dir):
+    cells = {}
+    # Get shapes.
+    polygon_input = Path(input_dir) / Path(tile_str + POLYGON_FILE_SUFFUX)
```

Comment:
```suggestion
    polygon_input = Path(input_dir) / f'{tile_str}{POLYGON_FILE_SUFFUX}'
```

I feel like issues with pathlib come up a lot in review. Everything that you do works, but it's more verbose that it needs to be: Path objects override `/`, so the second argument can just be sting.